docs/book/user-guide/advanced-guide/data-management/model-management.md

```diff
@@ -117,9 +117,14 @@ dashboard and API functionality to support the full ML lifecycle.
 
 Model versions are created implicitly as you are running your machine learning
 training, so you don't have to immediately think about this. If you want more
-control over how and when these versions are controlled, our API has you
+control over how and when these versions are controlled, 
+
+> our API has you
 covered, with options for whether new model versions are created as well as for
-the deletion of new model versions when pipeline runs fail. [See
+the deletion of new model versions when pipeline runs fail.
+
+is this still true? can't find these options in ModelVersion anymore.
```

Comment:
Nah, good catch. I would rephrase the whole paragraph into something like
```
Model versions are created implicitly as you are running your machine learning
training, so you don't have to immediately think about this. If you want more
control over versions, our API has you covered, with an option to explicitly
name your versions.
```

Reply:
got it!